src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App:React.FC = () => {
+  const [visibleTodos, setVisibleTodos] = useState([...todos]);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [errorExistence, setErrorExistence] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+
+  const titleTrim = title.trim().length;
```

Comment:
You can create it inside handleSubmit function, as you don't use it anywhere else